functions/int_select.h

```diff
@@ -0,0 +1,36 @@
+#ifndef FUNCTIONS_INT_SELECT_H
+#define FUNCTIONS_INT_SELECT_H
+
+// Usage: IntSelect<SELECTION, N>
```

Comment:
This needs to show that you can have as many integers as you like.

Reply:
updated